lib/Zonemaster/Backend/Config.pm

```diff
@@ -16,14 +16,23 @@ use Zonemaster::Backend::Validator qw( :untaint );
 use Zonemaster::Backend::DB;
 
 our $path;
+
+my @search_paths = (
+    '/etc/zonemaster/backend_config.ini',
+    '/usr/local/etc/zonemaster/backend_config.ini',
+    dist_file('Zonemaster-Backend', "backend_config.ini")
+);
+
 if ($ENV{ZONEMASTER_BACKEND_CONFIG_FILE}) {
     $path = $ENV{ZONEMASTER_BACKEND_CONFIG_FILE};
 }
-elsif ( -e '/etc/zonemaster/backend_config.ini' ) {
-    $path = '/etc/zonemaster/backend_config.ini';
-}
 else {
-    $path = dist_file('Zonemaster-Backend', "backend_config.ini");
+    for my $default_path (@search_paths) {
+        if ( -e $default_path ) {
+            $path = $default_path;
+            last;
+        }
```

Comment:
In the current code, `$path` is always defined. In the new code it will be undefined if the environment variable is undefinded and `backend_config.ini` does not exist in any of the three paths. That will probably give a different error message. It should probably be defined.

Reply:
The error will be the same actually. If `dist_file('Zonemaster-Backend', "backend_config.ini")` fails to find the file, then the backend fails to start with error `Failed to find shared file 'backend_config.ini' for dist 'Zonemaster-Backend'` in both cases, otherwise `$path` will be defined as the file exists.

Reply:
Please update the description of this PR to include the change for Travis, and I am fine with the changes.

Reply:
I've updated the description.